config/lava/chromeos/chromiumosvm-tast-template.jinja2

```diff
@@ -0,0 +1,66 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
+{% block main %}
+{{ super () }}
+{% endblock %}
+
+{% block actions %}
+{{ super () }}
+
+actions:
+- deploy:
+    images:
+      bios:
+        image_arg: -bios {bios}
+        url: http://storage.kernelci.org/images/uefi/edk2-stable202005/x86_64/OVMF.fd
+      rootfs:
+        image_arg: -drive format=raw,file={rootfs},id=lavatest2,if=none
+        url: http://storage.chromeos.kernelci.org/images/chromeos/chromiumos-amd64-generic/amd64/chromiumos_test_image.bin
+#        compression: gz
+    os: oe
+    timeout:
+      minutes: {{ job_timeout }}
+    to: tmpfs
+
+- boot:
+    method: qemu
+    media: tmpfs
+#    docker:
+#      image: kernelci/qemu
+#      binary: qemu-system-x86_64
+    timeout:
+      minutes: 5
+    prompts:
+    - "login:"
+
+- test:
```

Comment:
I thought this part would be moved to a `tast.jinja` template and shared between QEMU and Chromebook tests?

Reply:
Will do.

Reply:
Done